geode-core/src/main/java/com/gemstone/gemfire/management/internal/ManagementAgent.java

```diff
@@ -447,8 +449,7 @@ public synchronized void start() throws IOException {
       }
     };
 
-    String shiroConfig = this.config.getShiroInit();
-    if (! StringUtils.isBlank(shiroConfig) || isIntegratedSecurity()) {
+    if (isIntegratedSecurity()) {
```

Comment:
probably be easier to read if it just read "if (GeodeSecurityUtil.isJmxSecurityEnabled())" here.